src/executable-code/executable-fragment.js

```diff
@@ -314,11 +314,25 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
       ? document.body
       : this.nodes[0].querySelector(SELECTORS.CANVAS_PLACEHOLDER_OUTPUT);
   }
-    
+
+  removeImports (completeSnippet, importSnippet) {
+    let newArr = [];
+
+    completeSnippet.map(function(val) {
+      importSnippet.includes(val) ? '' : newArr.push(val);
+    });
+
+    importSnippet.map(function(val) {
```

Comment:
If you are mapping the complete snippet up there and only including each line in the `newArr` when it's not present in the `importSnippet`, what is the purpose of this second mapping?  Wouldn't be already our goal achieved after the first map?

Reply:
Thats true, thanks!